src/aws-cpp-sdk-core/include/aws/core/utils/base64/Base64.h

```diff
@@ -67,4 +67,3 @@ namespace Aws
         } // namespace Base64
     } // namespace Utils
 } // namespace Aws
-
```

Comment:
why? dont we want empty line at the end of files? lint is not flagging it?

Reply:
we should have a clang-tidy task as part of our CI, and i would happily add one. I would suggest we dont add one as part of this change though but a follow up task

Reply:
also i think the diff is messed up because im seeing a new line in the file locally

Reply:
probably the original file was created on windows, thus \r\n line ending has changed to just \n.